project/Build.scala

```diff
@@ -25,14 +26,15 @@ object MacroGLBuild extends MechaRepoBuild {
 
   val macroglScalaVersion = "2.11.8"
 
-  val lwjglVersion = "2.9.0"
+  val lwjglVersion = "2.9.3"
```

Comment:
I would change the LWJGL version in a separate PR, since its orthogonal to adding cross-compilation.